src/mobility/use-is-eligible-for-benefit.tsx

```diff
@@ -1,23 +1,35 @@
 import {useUserBenefitsQuery} from '@atb/mobility/queries/use-user-benefits-query';
 import {OperatorBenefitType} from '@atb-as/config-specs/lib/mobility-operators';
+import {isDefined} from '@atb/utils/presence';
+
+const benefitIdsRequiringValueCode: OperatorBenefitType['id'][] = [
+  'single-unlock',
+  'free-unlock',
+];
```

Comment:
I don't think we should reference operator benefit types in the code, as it breaks the low coupling between Firestore and the app code. Should get this configuration into Firestore instead.

Reply:
Yes, I should have thought about that! Will update

Reply:
https://github.com/AtB-AS/mittatb-app/pull/4490/commits/31a0831079d24389498f837a69b659b5c4fb2d1f